tests/tests/swfs/avm2/net_navigateToURL/Test.as

```diff
@@ -0,0 +1,34 @@
+package {
+    import flash.net.URLRequest;
+	import flash.net.URLVariables;
+	import flash.net.navigateToURL;
+	import flash.display.MovieClip;
+	public class Test extends MovieClip {
+        public function Test()
+        {
+            var request:URLRequest = new URLRequest("https://example.com/purchase/");
+			var variables:URLVariables = new URLVariables();
+            var TEST_PARAM_NAME:String = "sku";
+            var cases:Array = [
+                ["POST", "Test"],
+                ["POST", "sku=Test"],
+                ["GET", "Test"],
+                ["GET", "sku=TEST"]
+            ];
+            for each (var case_tuple:Array in cases) {
+                var method:String = case_tuple[0];
+                var data:String = case_tuple[1];
+                request.method = method;
+                if (data.indexOf(TEST_PARAM_NAME) != -1) { // pass data as string
+                    request.data = data;
+                } else { // pass data as object
+                    variables.sku = data;
+                    request.data = variables;
+                }
+                trace("// Method: ", method);
```

Comment:
Super nit, but the you remove the space at the end of the string to prevent the double spaced.

Reply:
Fixed in [cbc36f7](https://github.com/ruffle-rs/ruffle/pull/15129/commits/cbc36f794828b024c059eb86929dad8fe6e1fbcd)